phoenicis-repository/src/main/java/org/phoenicis/repository/dto/ApplicationDTO.java

```diff
@@ -53,12 +53,12 @@ private ApplicationDTO(Builder builder) {
         this.typeId = builder.typeId;
         this.categoryId = builder.categoryId;
         if (builder.id != null) {
-            if (builder.id.matches("^[a-zA-Z0-9_]+$")) {
+            if (builder.id.matches("^[a-zA-Z0-9_.]+$")) {
```

Comment:
Again we should think about moving this pattern to a static final field

Reply:
Again not sure where to put it. Also: this problem has not been introduced by this PR. In general, I agree that the current implementation is not the nicest one though.